### DISCUSSION TITLE
Using AufoFS to mount SeaweedFS in FUSE, I found that all file permissions are set to root

### DISCUSSION THREAD

**Discussion by hatef94** [seaweedfs/seaweedfs#5852] [Q&A]
Hi
I mainly used the materials provided [here](https://github-wiki-see.page/m/seaweedfs/seaweedfs/wiki/FUSE-Mount) to mount seaweed using AufoFS

```
cat /etc/auto.master

/-	/etc/auto.seaweed	--timeout=60
```

```
cat /etc/auto.seaweed

/weedfs -fstype=fuse,uid=hatef,gid=hatef,file_mode=0660,dir_mode=0770 :weedfsmount
```

```
cat weedfsmount

#!/bin/sh
sudo mount -t weed fuse /weedfs -o "filer='192.168.230.147:8888',filer.path=/"

```

it perfectly mounts the files in seaweedfs in `/weedfs` but the `uid` and `gid` are root
how can I change that to logged in user?

**Comment by chrislusf** [comment]
UID and GID are numbers.

**Comment by hatef94** [comment]
@chrislusf thank you for your response
I changed the content of `/etc/auto.seaweed` to this:
```
/weedfs -fstype=fuse,uid=1000,gid=1000,file_mode=0660,dir_mode=0770 :weedfsmount
```
still not successful
